platform/src/main/java/org/hillview/storage/CassandraDatabase.java

```diff
@@ -163,85 +218,79 @@ public CassTable(String keyspace, List<String> tables) {
             this.tables = tables;
         }
 
-        public String toString(){
-            StringBuilder result = new StringBuilder(" - " + this.keyspace);
+        public String toString() {
+            StringBuilder result = new StringBuilder(this.keyspace + ": [");
             for (String table : this.tables) {
-                result.append("\n\t").append(table);
+                result.append(" ").append(table);
             }
+            result.append("]\n");
             return result.toString();
         }
     }
 
-    /** The list of keyspaces and tables are obtained from cluster connection's metadata */
+    /**
+     * The list of keyspaces and tables are obtained from cluster connection's
+     * metadata
+     */
     private void setStoredTableInfo() {
         List<KeyspaceMetadata> keyspaces = Converters.checkNull(this.metadata).getKeyspaces();
-        String keyspace;
-        Collection<TableMetadata> tables;
-        List<String> tableNames;
         for (KeyspaceMetadata kMetadata : keyspaces) {
-            keyspace = kMetadata.getName();
-            tables = this.metadata.getKeyspace(keyspace).getTables();
-            tableNames = tables.stream().map(AbstractTableMetadata::getName)
-                .collect(Collectors.toList());
+            String keyspace = kMetadata.getName();
+            Collection<TableMetadata> tables = this.metadata.getKeyspace(keyspace).getTables();
+            List<String> tableNames = tables.stream().map(AbstractTableMetadata::getName).collect(Collectors.toList());
             this.cassTables.add(new CassTable(keyspace, tableNames));
         }
     }
 
-    /** Returns list of tables stored in the cluster  */
+    /** Returns list of tables stored in the cluster */
     public List<CassTable> getStoredTableInfo() {
         return this.cassTables;
     }
 
     /**
-     * Given the keyspace and table name, sstableutil will return the sstable-related files.
-     * Using sstableutil command is better than doing manually sstable search. Although we know
-     * the sstable parent dir from cassandra.yaml, we need to rule out the outdated sstables.
-     * And sstableutil is built specifically to identify which version are the most updated.
-      */
-    public String getSSTablePath() {
-        boolean isWindows = System.getProperty("os.name")
-            .toLowerCase().startsWith("windows");
-        Path ssTableUtilPath = Paths.get(ssTableUtilDir);
-        Path cassandraPath = Paths.get(info.cassandraRootDir);
+     * Given the keyspace and table name, sstableutil will return the
```

Comment:
in general the JavaDoc documentation of a method should just say *what* the method does, and not *how* it does it.
This kind of of comment is more suitable for the body of the method.